docset/winserver2022-ps/appx/Get-AppSharedPackageContainer.md

```diff
@@ -41,7 +41,7 @@ This command shows the packages in any shared package container that has a prefi
 ## PARAMETERS
 
 ### -AllUsers
-Get matching packages that are either deployed to any user or are provisioned to the machine.
+Unsupported. Will result in "A parameter cannot be found that matches parameter name" error.
 
```

Comment:
This is the error you get when the parameter does not exist. But on my Windows 11 computer, the parameter does exist.

```powershell
Get-Command Get-AppSharedPackageContainer, Remove-AppSharedPackageContainer -Syntax

Get-AppSharedPackageContainer [[-Name] <string>] [[-Id] <string>] [-AllUsers] [<CommonParameters>]

Remove-AppSharedPackageContainer [-Name] <string> [-ForceApplicationShutdown] [-AllUsers] [<CommonParameters>]
```

Are you running a different version of Appx where the parameter has been removed?

Reply:
It appears that the correct error message is "-AllUsers functionality is not yet implemented".

Reply:
I also get the not yet implemented message